### PR TITLE
add tikv-migration integration test 

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### CODE REVIEW

**Comment by zeminzhou** [comment]
@pingyu @haojinming PTAL~

File: tikv-migration/tikv-migration-verify-pipeline.yaml [hunk 1]
```diff
@@ -29,3 +29,21 @@ tasks:
       credentials:
         - jenkinsID: codecov-token-tikv-migration
           key: TIKV_MIGRATION_CODECOV_TOKEN
+
+    - taskName: "cdc-it"
```
Comment (pingyu) [inline tikv-migration/tikv-migration-verify-pipeline.yaml:33]: Suggest to use the name `cdc-integration-test`. `it` is not a well known abbreviation.

File: tikv-migration/tikv-migration-verify-pipeline.yaml [hunk 2]
```diff
@@ -29,3 +29,21 @@ tasks:
       credentials:
         - jenkinsID: codecov-token-tikv-migration
           key: TIKV_MIGRATION_CODECOV_TOKEN
+
+    - taskName: "cdc-it" 
+      checkerName: "it-check"
```
Comment (pingyu) [inline tikv-migration/tikv-migration-verify-pipeline.yaml:34]: There may not be such a checker. Suggest to ask @purelind for help.

Comment (zeminzhou) [reply tikv-migration/tikv-migration-verify-pipeline.yaml]: @purelind recommends using comon_check.

**Comment by zeminzhou** [comment]
@purelind PTAL~, thanks

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by purelind — APPROVED** [review]
LGTM

**Pull request merged**